python/ecs/ecs-service-with-advanced-alb-config/app.py

```diff
@@ -2,11 +2,11 @@
     aws_ec2 as ec2,
     aws_ecs as ecs,
     aws_elasticloadbalancingv2 as elbv2,
-    cdk,
+    core,
```

Comment:
We have been importing core types directly lately, so instead of core.Type it’s just Type. Not critical, but that’s what we are doing in TypeScript at least.